index.js

```diff
@@ -129,7 +186,17 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
+function getAllMoviesReleasedAtOrBeforeYear(movies, year) {
+  let released = [];
+  for (let i = 0; i < movies.length; i++){
+    newArr = movies[i].released.split(" ");
+    if (newArr[2] <= year){
+      released.push(movies[i]);
+    }
```

Comment:
I like the hard index to [2] but we should be careful when hard coding a value like this.  What if someone wrote a date differently?  We may get the wrong value